road/config.ini

```diff
@@ -1,32 +1,35 @@
-[DEFAULT]
-DBF_Files = LOADAM_FINAL.dbf, LOADMD_FINAL.dbf, LOADPM_FINAL.dbf ,LOADEV_FINAL.dbf, LOADEA_FINAL.dbf
+[DBF]
+
 DBF_Directory = Q:/Model Development/Simwrapper Validation/notebooks/roads/Data
+DBF_Files = LOADAM_FINAL.dbf, LOADMD_FINAL.dbf, LOADPM_FINAL.dbf ,LOADEV_FINAL.dbf, LOADEA_FINAL.dbf
 DBF_Column_Names = A, B, AT, FT, V_1
 
+[OBSERVED FILE]
 Excel_File_Path = Q:\Model Development\Simwrapper Validation\templates\Road_Validation_Draft_v2_Run17.xlsx
 Sheet_Name = LocsData
-Output_File_Name = output.csv
+Obs_usecols = K,L,V:AA
 Excel_Extra_Columns = Loc Type, A, B, Source ID, AT Grp,FT Grp, Location, Road, Dir
-Obs_usecols = V:AA
 
+[SCATTER INPUT]
 Chosen_period = Daily
 Classification_col = Loc Type
 Combined_DF_Cols = Loc Type, A, B, AT, FT, Source ID, AT Grp, FT Grp, Location, Road, Dir
-; Selected_cols= Loc Type, A, B,AT,FT, Estimated Volumn, Observed Volumn, Errors, Squared Erros, Percent Errors
 
-[PLOT1]
+[EST SCATTER PLOT]
 XField = Observed Volume
 YField = Estimated Volume
 Fields = Observed Volume, Estimated Volume, Loc Type, A, B, Location, Road, Dir
 NominalFields = Loc Type, A, B, Location, Road, Dir
 Name = Estimation Volume
 
-[PLOT2]
+[Perc SCATTER PLOT]

```

Comment:
what's 'perc'? add a comment (toml also allows inline comments so you can add that as a `[foo_bar]  # lalala` on this line too

Reply:
it's percent, I will revise it